optuna/testing/pytest_storages.py

```diff
@@ -741,6 +744,8 @@ def test_get_all_trials_state_option(self, storage: BaseStorage) -> None:
         for state in states:
             t = _generate_trial(generator)
             t.state = state
+            if t.state == TrialState.COMPLETE:
+                t.values = [0.0]
```

Comment:
```suggestion
            if t.state == TrialState.COMPLETE:
                t.values = [0.0]
                t.value = None
```

How about setting `t.value` to `None`, since the `value` and `values` attributes must not be specified in the same time? Currently, the `value` attribute is settled in `_generate_trial`, when the state is already `TrialState.COMPLETE`.

https://github.com/optuna/optuna/blob/21ac00e3ed4a51adf2fd5942fa8adc909b82f19c/optuna/trial/_frozen.py#L159-L160